airbyte-integrations/connectors/source-bamboo-hr/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 90916976-a132-4ce9-8bce-82a03dd58788
-  dockerImageTag: 0.3.5
+  dockerImageTag: 0.3.6
```

Comment:
You're adding a new stream and parameter, please bump the image to `0.4.0`. Don't forget to run `poetry lock` inside the bamboo connector folder to generate the poetry lock file.